apps/server/src/shared/repo/task/task.repo.integration.spec.ts

```diff
@@ -1604,4 +1604,18 @@ describe('TaskRepo', () => {
 			expect(foundTask.name).toEqual('important task');
 		});
 	});
+
+	describe('delete', () => {
+		it('should remove a task in the database', async () => {
+			const task = taskFactory.build({ name: 'important task' });
```

Comment:
Just a note: It's not necessary to provide a custom name if you don't want. But you likely know that :-)